clients/js/src/examples/multisig.ts

```diff
@@ -1,6 +1,7 @@
 import {
   address,
   appendTransactionMessageInstructions,
+  assertIsTransactionWithinSizeLimit,
```

Comment:
`assertIsSendableTransaction` is slightly more robust because if we ever add more constraints to what a `SendableTransaction` means, you won't have to change anything.